gradle/libs.versions.toml

```diff
@@ -1,13 +1,13 @@
 [versions]
-biz-aQute-bnd = "6.1.0"
-checkStyle = "9.2"
+biz-aQute-bnd = "6.2.0"
+checkStyle = "9.3"
 com-squareup-moshi = "1.13.0"
 com-squareup-okio = "3.0.0"
 de-mannodermaus-junit5 = "1.3.0"
 graalvm = "22.0.0.2"
 org-bouncycastle = "1.70"
 org-conscrypt = "2.5.2"
-org-jetbrains-kotlin = "1.6.10"
+org-jetbrains-kotlin = "1.6.20-RC"
```

Comment:
Will revert this, or potentially wait for the release before landing.  Using it to test earlier for now.